cloudml-template/README.md

```diff
@@ -0,0 +1,24 @@
+# TensorFlow Trainer Package Template 
```

Comment:
@ksalama : It seems as if this could become a good template for users who want to deploy jobs involving the canned estimators to ML Engine. There are a couple of things that I think are missing for this purpose:

1. Guidance on submitting modified versions of this template (using `config.yaml`) to ML Engine

1. An example which shows users how to turn this template into a working job. There is a lot of code here, and although the table in the README is very helpful, someone using the template will still have to spend some time reading through the code, as well.

Although I have not tried using it for this purpose, I have a feeling that the template does not adapt well to the task of setting up training jobs with custom estimators or even with hand-written training loops with input data potentially requiring some preprocessing, being zipped from multiple sources, etc. In general, keeping this template up-to-date with the changes tensorflow is making to its input pipelines seems like it would be a lot of work and, if we are pitching it as a general template for the submission of training jobs to ML Engine, this is work we would have to do.

For that reason, I suggest not presenting this as a generic trainer template, but rather as a template for canned estimator usage or for common data science problems.